### ISSUE TITLE
Charts aren't accessible

### ISSUE THREAD

**Issue by Changaco** [liberapay/liberapay.com#1871]
Text descriptions should be added to the charts in [/about/stats](https://liberapay.com/about/stats) and other pages. Ideally the text description would convey what can be seen by looking at the chart, for example whether it's currently stable or going upwards or downwards. These descriptions should be generated server-side since they will have to be translated.

**Comment by rkingett** [comment]
Could there be an option to review this data in a native HTML table? For people who are legally blind and rely on screen reader software, it's ideal to provide a text alternative for your data.

A text alternative for a chart or graph is usually made available using an HTML table, outputting the data in a tabular structure. What this does is allow someone using assistive technology to gain an understanding of the data without relying on the visual chart or graph.
